package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Node.js GitHub Bot",
   "scripts": {
     "start": "node server.js | bunyan -o short",
-    "test": "STATUS=0; tap test/**/*.test.js || STATUS=$?; standard || STATUS=$?; exit $STATUS",
+    "test": "STATUS=0; tap --no-check-coverage test/**/*.test.js || STATUS=$?; standard || STATUS=$?; exit $STATUS",
     "test:watch": "nodemon -q -x 'npm test'"
   },
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 12.0.0"
```

Comment:
This surely makes sense, not only having packages in mind, but also on the server where we're running 14.x.